### PR TITLE
[skip ci] Update api description

### PR DESCRIPTION
**Pull request by ireneontheway5**
Update api description

### CODE REVIEW

File: doc/source/api.rst [hunk 1]
```diff
@@ -37,27 +37,27 @@ Methods
...
+| `bulk_insert() <#milvus.Milvus.bulk_insert>`_                        | Inserts columnar entities to a specified collection.                   |
 +----------------------------------------------------------------------+------------------------------------------------------------------------+
 | `insert() <#milvus.Milvus.insert>`_                                  | Inserts entities to a specified collection.                            |
 +----------------------------------------------------------------------+------------------------------------------------------------------------+
 | `get_entity_by_id() <#milvus.Milvus.get_entity_by_id>`_              | Retrieves entities by entity ID.                                       |
 +----------------------------------------------------------------------+------------------------------------------------------------------------+
-| `list_id_in_segment() <#milvus.Milvus.list_id_in_segment>`_          | Obtain the list of ids in specified segment.                           |
+| `list_id_in_segment() <#milvus.Milvus.list_id_in_segment>`_          | Obtains the list of ids in specified segment.                           |
```
Comment (ireneontheway5) [inline doc/source/api.rst:46]:
```suggestion
| `list_id_in_segment() <#milvus.Milvus.list_id_in_segment>`_          | Obtains the list of ids in a specified segment.                           |
```

**Comment by ireneontheway5** [comment]
Can you help me review this documents? @YiyunNi @BossZou

**Review by BossZou — APPROVED** [review]

**Comment by YiyunNi** [comment]
Could you please replace all "retrieves" and "obtains" with "Gets", "Drops" with "removes" for the sake of consistency? @ireneontheway5 @PahudPlus

**Comment by ireneontheway5** [comment]
> Could you please replace all "retrieves" and "obtains" with "Gets", "Drops" with "removes" for the sake of consistency? @ireneontheway5 @PahudPlus

Done in https://github.com/milvus-io/pymilvus/pull/387

**Pull request merged**